### DISCUSSION TITLE
Limitierung verhält sich seltsam

### DISCUSSION THREAD

**Discussion by frontlinerde** [tbnobody/OpenDTU#595] [Q&A] [answered]
Hi,
hab aktuell ein interessantes Phänomen mit meiner openDTU.
Bis gestern war ein HM-600 angeschlossen und den hab ich mal testhalber durch nen HM-1500 ersetzt.
der HM-600 hat problemlos funktioniert und ich hab ihn aus der DTU gelöscht.
Leider übernimmt er die von mir per MQTT gesendeten Limits nicht korrekt.
Aktuell könnte er mehr leisten und beschränkt aber auf 90W was wiederum ungefähr 14% von 600 sind.
Die 14% entsprechen aber wiederum ungefähr den 203Watt für nen HM-1500
scheinbar hat er sich irgendwo den HM-600 gemerkt oder liegt es daran, dass aktuell nur 2 Panele angeschlossen sind?
Jemand ne Idee?
Danke

![331731532_972730707049491_7226637893553238448_n](https://user-images.githubusercontent.com/108725894/219363433-8ae4380e-85fd-4360-a297-c0f5596d5b58.jpg)

**Comment by tbnobody** [comment, marked as answer]
> oder liegt es daran, dass aktuell nur 2 Panele angeschlossen sind?

Korrekt. Das Limit wird erstmal gleichmäßig auf die Ports 1&2 sowie 3&4 aufgeteilt. (Das ist leider bekannte Einschränkung des Inverters) Wenn du deine Panels an 1 und 3 anschließt sollte es sich so verhalten wie von dir gewünscht. (Oder du rechnest den unterschied entsprechend in dein vorgegebenes Limit mit ein)

**Reply by frontlinerde** [reply]
ah okay,
vielen Dank
